src/components/CreatePollForm.vue

```diff
@@ -0,0 +1,304 @@
+<template>
+  <form @submit.prevent="createPoll()" id="poll_data_form">
+    <div class="input-section">
+      <label class="dialog-item" for="poll_title">Заголовок: </label>
+      <Textarea
+        class="input-poll"
+        id="poll_title"
+        placeholder="Заголовок"
+        rows="3"
+        v-model.trim="pollData.title"
+        :class="{
+          'p-invalid': v$.pollData.title.$error,
+        }"
+        @input="v$.pollData.title.$touch"
+      />
+      <small v-if="v$.pollData.title.$error" id="poll_title" class="p-error">{{
+        v$.pollData.title.$errors[0].$message
+      }}</small>
+    </div>
+    <div class="input-section">
+      <label class="dialog-item" for="poll_description">Опис: </label>
+      <Textarea
+        id="poll_description"
+        placeholder="Детальний опис"
+        rows="8"
+        v-model.trim="pollData.description"
+        :class="{
+          'p-invalid': v$.pollData.description.$error,
+        }"
+        @input="v$.pollData.description.$touch"
+      />
+      <small v-if="v$.pollData.description.$error" id="poll_description" class="p-error">{{
+        v$.pollData.description.$errors[0].$message
+      }}</small>
+    </div>
+    <div class="input-section">
+      <label class="dialog-item" for="poll_polledHouses">Список будинків: </label>
+      <div class="checkbox-section">
+        <div v-for="house of houses" :key="house.id" class="p-field-checkbox">
+          <Checkbox name="category" :id="house.id" :value="house" v-model="pollData.polledHouses" />
+          <label class="house-label" :for="house.id">
+            {{ house.address?.houseNumber }}, {{ house.address?.houseBlock }}, {{ house.address?.district }},
+            {{ house.address?.street }}
```

Comment:
please consider creating a computed property for this

it will improve readability, performance and this section can be reusable